openpype/hosts/blender/blender_addon/addons/render_playblast.py

```diff
@@ -0,0 +1,190 @@
+                # Set PNG specific settings for transparency
+                scene.render.image_settings.color_mode = "RGBA"
+                scene.render.film_transparent = True
+                scene.render.engine = "CYCLES"
+            else:
+                # set color mode to RGB (no transparency support)
+                scene.render.image_settings.color_mode = "RGB"
+                scene.render.film_transparent = False
+
+            # Apply container settings for ffmpeg if needed
+            container = options.get('container')
+            if container:
+                scene.render.ffmpeg.format = container
+
+            logging.info(f"{'Camera view' if scene.playblast_settings.use_camera_view else 'Viewport'} rendering at: {scene.render.filepath}")
+            result = bpy.ops.render.opengl(animation=True)
+            if result != {"FINISHED"}:
+                logging.error(f"Error rendering with file_format {file_format} using OpenGL")
+                break
+
+        # Restore the original settings
+        scene.render.filepath = render_filepath
+        scene.render.image_settings.file_format = file_format
+        scene.render.use_file_extension = file_extension_use
+
+        # Restore color_mode safely: check if the original color_mode is allowed
+        if color_mode in ['RGB', 'BW'] or (file_format == "PNG" and color_mode == "RGBA"):
+            scene.render.image_settings.color_mode = color_mode
+        else:
+            # Fallback to RGB if the original mode is not compatible with the format
+            scene.render.image_settings.color_mode = 'RGB'
+
+        if region and scene.playblast_settings.use_camera_view:
+            region.view_perspective = perspective_region
+
+        if scene.playblast_settings.use_transparent_bg:
+            # reset to memorized parameters for render
+            scene.render.engine = engine
+            scene.render.film_transparent = film_transparent
+
+        return {"FINISHED"}
+
+    def get_view_3D_region(self):
+        """Find the VIEW_3D region and return its region_3d space."""
+        for area in bpy.context.screen.areas:
+            if area.type == 'VIEW_3D':
+                for space in area.spaces:
+                    if space.type == 'VIEW_3D':
+                        return space.region_3d
+        return None
+
+
+class OBJECT_OT_OPEN_PLAYBLAST_FOLDER(bpy.types.Operator):
+    bl_idname = "playblast.open"
+    bl_label = "Open Last Playblast Folder"
+
+    def execute(self, context):
+        # Get the path to the most recent playblast folder
+        latest_playblast_filepath = get_path_from_template('playblast',
+                                                           'folder')
+
+        if not os.path.exists(latest_playblast_filepath):
+            self.report({'ERROR'}, f"File '{latest_playblast_filepath}' not found")
+            return {'CANCELLED'}
+
+        if 'win' in sys.platform:  # windows
```

Comment:
Just like the other comment below, the code should be moved into a function stored in `openpype/lib/execute.py`

Reply:
Fixed in my last commit